packages/@angular/cli/utilities/completion.sh

```diff
@@ -6,11 +6,10 @@
 #           or  ng completion 1>> ~/.zshrc 2>>&1
 #
 
-ng_opts='b build completion doc e2e g generate get github-pages:deploy gh-pages:deploy h help i init install lint make-this-awesome new s serve server set t test v version'
+ng_opts='b build completion doc e2e g generate get h help i init install lint make-this-awesome new s serve server set t test v version'
```

Comment:
Could you undo the changes to completion.sh? We have a change coming to it and this won't be needed, and I'd like to commit both without having to rebase one of you ;)